openreview/venue_request/process/matching_pre_process.py

```diff
@@ -30,3 +30,9 @@ def process(client, note, invitation):
         _, num_submissions = client_v2.get_notes(content={ 'venueid':submission_venue_id }, limit=1, with_count=True)
         if compute_affinity_scores and num_submissions >= 2000:
             raise openreview.OpenReviewException(f'Can not compute affinity scores for venues with 2000+ papers. Please contact us at info@openreview.net to compute your scores.')
+
+        area_chairs_name = domain.get_content_value('area_chairs_name')
+        if compute_conflicts and senior_area_chairs_name and area_chairs_name and matching_group.endswith(area_chairs_name):
```

Comment:
you can get the area_chairs_id from the domain content to compare with the matching_group instead of using endswith that could match with any group like Senior_Area_Chairs, Area_Chairs, etc.

Reply:
in line 24 you can do the same with senior_area_chairs_id.

Reply:
Fixed